docs/cloud.rst

```diff
@@ -1,37 +1,4 @@
 Cloud
```

Comment:
@tzach can we leave those files alone ? and new one, and point to them ? to just remove those from being referenced in the index ? like we did in the README.rst ?

I don't want to hit those on each merge...

Reply:
What I can do is exclude them from building in conf.py
Make sense?

Reply:
yes, lets not touch files that we don't want

Reply:
For Cloud, we will soon add Scylla Cloud data, so this change does make sense

Reply:
Even for cloud, let's have a different filename...

Reply:
I made many other changes in docs files.
We can move the docs to a new dir, and ignore current docs.